core/src/Platforms/iOS/EmbeddedWebview/AuthenticationAgentUIViewController.cs

```diff
@@ -64,90 +63,93 @@ public override void ViewDidLoad()
 
             View.BackgroundColor = UIColor.White;
 
-            webView = new UIWebView(View.Bounds);
-            webView.ShouldStartLoad = (wView, request, navType) =>
+            WKWebViewConfiguration webConfiguration = new WKWebViewConfiguration();
+
+            wkWebView = new WKWebView(View.Bounds, webConfiguration);
+            NSUrl url = wkWebView.Url;
+
+            wkWebView.LoadRequest(new NSUrlRequest(url));
```

Comment:
>wkWebView.LoadRequest(new NSUrlRequest(url)); [](start = 12, length = 45)

perhaps the actual question is.... why is this to be loaded into the webview?

Reply:
I'll look into this. Was trying to keep what we had, but i'll go back and double check these things you brought up.